lib/stack_master/commands/nag.rb

```diff
@@ -0,0 +1,32 @@
+module StackMaster
+  module Commands
+    class Nag
+      include Command
+      include Commander::UI
+
+      def perform
+        if proposed_stack.template_format == :yaml
+          failed! 'cfn_nag doesn\'t support yaml formatted templates.'
```

Comment:
Can we do a temporary file with a yaml -> json conversion and run the test on it?

Reply:
Also, cfn_nag claims they support yaml: https://github.com/stelligent/cfn_nag/blob/master/lib/cfn-nag/cfn_nag.rb#L76

Reply:
You're right!  Tested and works; fixed in d3789e4.